src/Resources/views/CRUD/edit_orm_many_association_script.html.twig

```diff
@@ -339,6 +379,27 @@ This code manage the many-to-[one|many] association field popup
 
         return false;
     }
+    
+    {#
+        This code is used to defined the "edit" popup
```

Comment:
"define"

Reply:
ah, it was a copy paste, I guess all comments in that file need a refresh.